### DISCUSSION TITLE
Best way to backup the booru?

### DISCUSSION THREAD

**Discussion by crowscrows** [rr-/szurubooru#581] [Q&A] [answered]
What would be a simple way to make backups of the booru? Keeping in all the tags, images and whatnot. I could just drag the docker folders into another hard drive, but then I'm not sure how I would re-load those backups if anything happened

**Comment by dyedfire** [comment, marked as answer]
As far as I'm aware, all stats are kept in the postgres sql library. I can't speak for how you're building it but from my experience, the .env file determines this location:
`MOUNT_SQL=/mnt/cache/appdata/sql_stash`

The raw data itself (.png .jpg .mp4) are stored in a separate location:

`MOUNT_DATA=/mnt/user/Examplefolder/data`

Don't know if that helps but that's all I know
